src/asset-selector/AssetSelectorItem.js

```diff
@@ -27,14 +27,9 @@ export default class AssetSelectorItem extends React.Component {
 
 	render() {
 		const { asset, onSelect, isFavorite, isSelected } = this.props;
-		const focuser = node => {
-			if (node !== null && isSelected) {
-				node.focus();
-			}
-		};
 
 		return (
-			<tr autoFocus={isSelected} tabIndex={0} ref={focuser} >
+			<tr className={isSelected ? 'focus-like' : ''} tabIndex={0}>
```

Comment:
tr:focus break input focus ~
when user typed on asset selector query box, having tr:focus will defocus the input box ~
and I notice we just want to show user what is selected, instead of having focus there